<ui version="4.0" >
 <class>QSerialPortTerminal</class>
 <widget class="QMainWindow" name="QSerialPortTerminal" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>289</height>
   </rect>
  </property>
  <property name="font" >
   <font>
    <family>Wenquanyi Bitmap Song</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle" >
   <string>QSerialPortTerminal</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QHBoxLayout" name="horizontalLayout" >
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <widget class="QGroupBox" name="groupBoxSettings" >
        <property name="title" >
         <string>设置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout" >
         <item row="0" column="0" >
          <widget class="QLabel" name="labelPort" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>端口 :</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QComboBox" name="comboBoxPortName" />
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="labelBaudRate" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>波特率 :</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QComboBox" name="comboBoxBaudRate" />
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="labelData" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>数据位 :</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QComboBox" name="comboBoxDataBits" />
         </item>
         <item row="3" column="0" >
          <widget class="QLabel" name="labelParity" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>校验位 :</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" >
          <widget class="QComboBox" name="comboBoxParity" />
         </item>
         <item row="4" column="0" >
          <widget class="QLabel" name="labelStop" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>停止位 :</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1" >
          <widget class="QComboBox" name="comboBoxStopBits" />
         </item>
         <item row="5" column="0" >
          <widget class="QLabel" name="labelFlowControl" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>流控制 :</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1" >
          <widget class="QComboBox" name="comboBoxFlowControl" />
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxOptions" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="title" >
         <string>选项</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2" >
         <property name="topMargin" >
          <number>4</number>
         </property>
         <property name="bottomMargin" >
          <number>4</number>
         </property>
         <property name="horizontalSpacing" >
          <number>5</number>
         </property>
         <property name="verticalSpacing" >
          <number>4</number>
         </property>
         <item row="0" column="0" >
          <widget class="QLabel" name="labelCodec" >
           <property name="text" >
            <string>编码 :</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QComboBox" name="comboBoxCodecs" />
         </item>
         <item row="1" column="1" >
          <spacer name="verticalSpacer" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="frame_2" >
      <property name="frameShape" >
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>4</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>9</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QPushButton" name="pushButtonOpen" >
           <property name="text" >
            <string>打开端口</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonClose" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>关闭端口</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditReceive" >
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLineEdit" name="lineEditSend" />
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxCR" >
           <property name="text" >
            <string>CR</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxLF" >
           <property name="text" >
            <string>LF</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSend" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>textEditReceive</zorder>
      <zorder>pushButtonSend</zorder>
      <zorder></zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
